src/static/js/earn.js

```diff
@@ -1183,26 +1298,25 @@ async function main() {
     if ( options.display_amount > 0 ) {
       has_options = true
       approveBtn = `<button data-btn="${options.approve}" class="btn btn-sm mx-10 approveBtn" ><ion-icon name="bag-check-outline" role="img" class="md hydrated" aria-label="bag check outline"></ion-icon> Approve</button>`;
-      stakeBtn = `<button data-btn="${options.stake}" class="btn btn-sm mx-10 btn-success stakeBtn"><ion-icon name="lock-open-outline"></ion-icon> Stake S3F</button>`;
+      stakeBtn = `<button data-btn="${options.stake}" class="btn btn-sm mx-10 btn-success stakeBtn"><ion-icon name="lock-open-outline"></ion-icon> Stake PGL</button>`;
     }
-    if ( options.staked_pool / 1e18 > 0 ) {
+    if ( options.staked_pool.amount / 1e18 > 0 ) {
```

Comment:
Same here with the condition

Reply:
They come from different functions. The "green" is from poolSNOB(), the "red" is from poolS3F"